docs/UserGuide.md

```diff
@@ -408,7 +408,7 @@ Changes the theme of the app.
 
 Format: `theme THEME`
 
-#####Parameters:
+##### Parameters:
```

Comment:
I think this will cause a problem in the UG.
My PR also faced the same issue, so I had to [fix it](https://github.com/AY2122S1-CS2103T-F12-1/tp/pull/241)